Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Beckn/Confirm.hs

```diff
@@ -112,6 +115,7 @@ handler transporter req quote = do
       case quote of
         Left (driver, driverQuote) -> do
           ride <- buildRide driver.id booking
+          createAndTriggerRideEvent ride.id ride.status ride.createdAt ride.updatedAt driver.id transporter.id RideCreated
```

Comment:
Lets pass the whole `ride` and `person` object from here. Something like this
`createAndTriggerRideEvent RideCreated {ride, driver}`